src/main/java/com/tvtracker/enterprise/dto/MediaEntry.java

```diff
@@ -11,7 +11,6 @@
      */
     private int entryId;
     private String title;
-    private String username;
```

Comment:
I removed the username variable as it is already included in the UserAccess dto. I am unclear whether MediaEntry has a username that is completely separate from the UserAccess username or if this is meant to act as a foreign key. If the latter, this should be a unique primary key value such as, user_id. 
If this variable is needed, I suggest using a more DTO specific variable name to reduce confusion and technical debt, as found here https://martinfowler.com/bliki/TechnicalDebt.html